src/lib/schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -53,6 +54,12 @@ import { Observable } from 'rxjs';
   changeDetection: ChangeDetectionStrategy.<%= changeDetection %><% } %>
 })
 export class <%= classify(name) %>Component {
-  isHandset$: Observable<BreakpointState> = this.breakpointObserver.observe(Breakpoints.Handset);
+
+  isHandset$: Observable<BreakpointState> = this.breakpointObserver.observe(Breakpoints.Handset)
```

Comment:
This Observable has been changed to a boolean.

Reply:
There has already been a PR to fix it.

Reply:
Apologies, I updated to latest and the issue was still present. Found the PR which fixes it now. 👍